hasura.planx.uk/migrations/1683197635599_delete_payment_request_events_on_paid/down.sql

```diff
@@ -0,0 +1,2 @@
+DROP TRIGGER IF EXISTS delete_payment_request_scheduled_email_events_trigger ON payment_requests;
```

Comment:
not that I ever expect this down migration to run, but I think because there's a `COMMENT` on the trigger this would technically fail without a `CASCADE` to drop the corresponding comment as well ?

Reply:
Good thinking but [comments are automatically dropped when their object is dropped](https://www.postgresql.org/docs/current/sql-comment.html#:~:text=Comments%20are%20automatically%20dropped%20when%20their%20object%20is%20dropped). I did run some up/down commands and both work fine :+1: